docker-compose.yaml

```diff
@@ -1,14 +1,13 @@
 services:
   nuts-node1:
     image: nutsfoundation/nuts-node:6.0.0-rc.3
-    environment:
-      NUTS_CONFIGFILE: /opt/nuts/config/nuts.yaml
+#    user: when running on linux, this must be set to the host user if the datadir is mounted ($USER:$USER or $UID:$UID depending on what env variables exist)
     ports:
       - 5551:5551
       - 1323:1323
     volumes:
-      - "./config/node1:/opt/nuts/config:ro"
-      - "./data/node1:/opt/nuts/data:rw"
+      - "./config/node1:/nuts/config:ro"
+      - "./data/node1:/nuts/data:rw"
```

Comment:
The data dir is currently set to `/nuts/data`, so if this dir needs to be mounted, either accept this change or add `datadir: /opt/nuts/data` to the config